### PR TITLE
feat: [SRM-9813]: doing hotfix for infinite calls on service health page

### PR DESCRIPTION
**Pull request by harshilbits**
##### Jira Links:
1. Doing hotfix for infinite calls on service health page

https://harness.atlassian.net/browse/SRM-9813
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Review by deepesh-ui — DISMISSED** [review]

**Review by kapilkumawat86 — DISMISSED** [review]

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/HealthScoreChart/HealthScoreChart.tsx [hunk 1]
```diff
@@ -26,22 +26,36 @@ export default function HealthScoreChart(props: HealthScoreChartProps): JSX.Elem
...
     identifier: monitoredServiceIdentifier,
-    queryParams: {
-      accountId,
-      orgIdentifier,
-      projectIdentifier,
-      duration: duration?.value as TimePeriodEnum,
-      endTime: endTime || Date.now()
-    }
+    queryParams,
+    lazy: true
```
Comment (vkbansal) [inline src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/HealthScoreChart/HealthScoreChart.tsx:47]: You don't need lazy now

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/HealthScoreChart/HealthScoreChart.tsx [hunk 2]
```diff
@@ -26,22 +26,36 @@ export default function HealthScoreChart(props: HealthScoreChartProps): JSX.Elem
   const { orgIdentifier, projectIdentifier, accountId } = useParams<ProjectPathProps>()
   const [seriesData, setSeriesData] = useState<ColumnData[]>([])
 
+  const queryParams = useMemo(() => {
+    return {
+      accountId,
+      projectIdentifier,
+      orgIdentifier,
+      duration: duration?.value as TimePeriodEnum,
+      endTime: endTime || Date.now()
```
Comment (abhinavrastogi-harness) [inline src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/HealthScoreChart/HealthScoreChart.tsx:35]: only `Data.now()` needs to be memoized.. other params wont change.

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/HealthScoreChart/HealthScoreChart.tsx [hunk 3]
```diff
@@ -26,22 +26,36 @@ export default function HealthScoreChart(props: HealthScoreChartProps): JSX.Elem
...
-      orgIdentifier,
-      projectIdentifier,
-      duration: duration?.value as TimePeriodEnum,
-      endTime: endTime || Date.now()
-    }
+    queryParams,
+    lazy: true
   })
 
+  useEffect(() => {
```
[the remark this answers is not in the document]
Comment (abhinavrastogi-harness) [reply src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/HealthScoreChart/HealthScoreChart.tsx:50]: +1

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/HealthScoreChart/HealthScoreChart.tsx [hunk 4]
```diff
@@ -26,22 +26,36 @@ export default function HealthScoreChart(props: HealthScoreChartProps): JSX.Elem
...
     loading: healthScoreDataWithMSIdentifierLoading,
     error: healthScoreDataWithMSIdentifierError
   } = useGetMonitoredServiceOverAllHealthScore({
     identifier: monitoredServiceIdentifier,
-    queryParams: {
-      accountId,
-      orgIdentifier,
-      projectIdentifier,
-      duration: duration?.value as TimePeriodEnum,
-      endTime: endTime || Date.now()
```
Comment (vkbansal) [inline src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/HealthScoreChart/HealthScoreChart.tsx:41]: This was the main culprit for infinite calls

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/HealthScoreChart/HealthScoreChart.tsx (hunk 2 above)
Comment (harshilbits) [reply src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/HealthScoreChart/HealthScoreChart.tsx:35]: done, duration can also change as we have the option in UI to change the duration from dropdown.

Removed  accountId, projectIdentifier, orgIdentifier,

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/HealthScoreChart/HealthScoreChart.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (harshilbits) [reply src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/HealthScoreChart/HealthScoreChart.tsx:49]: done.

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/HealthScoreChart/HealthScoreChart.tsx (hunk 1 above)
Comment (harshilbits) [reply src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/HealthScoreChart/HealthScoreChart.tsx]: done.

**Review by vkbansal — APPROVED** [review]

**Review by praneshg239 — APPROVED** [review]

**Review by deepesh-ui — APPROVED** [review]

**Comment by harshilbits** [comment]
trigger cypress

**Pull request closed without merging**